### ISSUE TITLE
Update draft according feedback received during the call for adoption

### ISSUE THREAD

**Issue by youennf** [sframe-wg/sframe#70]
Some requests for change have been received during the call for adoption, in particular:
- Bernard Aboba feedback at https://mailarchive.ietf.org/arch/msg/sframe/G3jC1bqZNqYz0dFr9JBOqni1lB0/
- Eric Rescorla feedback at https://mailarchive.ietf.org/arch/msg/sframe/yWMP6BRT_SGlhjTiaOmgfBVTVmw/

We should split this feedback in manageable issues.

**Comment by martinthomson** [comment]
@IETF 116 we all decided that @youennf can do some work here

**Comment by bifurcation** [comment]
I reviewed the linked messages from Bernard and EKR.  They are mostly overcome by events.

Bernard's message is focused on use cases: At the time of adoption,  the document had on the one hand, lots of RTP specifics baked in, and on the other hand, aspirations of being more general.  Since adoption, we have stripped things down so that the document describes a pure encapsulation format, with any RTP specifics clearly delineated as examples.  So I think these concerns have been addressed.

EKR had a few more detailed points, most of which have already been addressed.  I filed the remainder as #142.
